CRFModuleValidation/Cardio_12MT.json

```diff
@@ -57,41 +85,45 @@
+                                               "actions"        : { "goForward": { "iconName": "runStartButton" }},
+                                               "viewTheme"      : { "viewIdentifier": "ActiveInstruction", "storyboardIdentifier": "ActiveTaskSteps" }
                                                },
                                                {
                                                "identifier"   : "countdown",
                                                "type"         : "active",
                                                "text"         : "Start in",
                                                "shouldHideActions": ["goBackward", "goForward"],
                                                "duration": 5,
-                                               "commands": ["playSound", "transitionAutomatically"]
+                                               "commands": ["transitionAutomatically"],
+                                               "viewTheme"      : { "viewIdentifier": "Countdown", "storyboardIdentifier": "ActiveTaskSteps" }
                                                },
                                                {
                                                "identifier"   : "run",
                                                "type"         : "active",
-                                               "imageBefore"  : "12MinFullScreen",
+                                               "image"        : "12MinFullScreen",
                                                "shouldHideActions": ["goBackward", "goForward"],
                                                "duration": 12,
-                                               "commands": ["playSound", "transitionAutomatically"]
+                                               "commands": ["playSound", "vibrate", "transitionAutomatically"]
                                                }
                                                ]
                            },
                            {
                            "identifier"         : "heartRate.after",
-                           "type"               : "heartRate",
-                           "title"              : "Stand still for 1 minute",
-                           "subtitle"           : "Almost done! Stand still for a minute to measure your heart rate recovery.",
-                           "detail"             : "Your heart rate changed to",
-                           "taskTransformer"    : { "resourceName": "HeartrateStep.json"}
+                           "type"               : "transform",
+                           "replacementSteps"   : [{ "identifier"   : "instruction",
+                                                   "title"        : "Stand still for 1 minute",
+                                                   "text"         : "Almost done! Stand still for a minute to measure your heart rate recovery." },
```

Comment:
Are they supposed to be doing the finger-over-camera-and-flash thingy again here? If so, the instructions should maybe mention it? If not, where are we getting the heart rate from for the feedback?

Reply:
The instructions are also included in the actual heart rate step... but not yet. (cuz I haven't done that step yet).